diff -aur a/bstack.c b/bstack.c
--- a/bstack.c	2010-08-18 01:17:14.608668919 -0400
+++ b/bstack.c	2010-08-18 01:21:06.408124981 -0400
@@ -9,6 +9,7 @@
 		return;
 	/* master */
 	c = nexttiled(m->clients);
+	c->bw = (n > 1 ? borderpx : 0);
 	mh = m->mfact * m->wh;
 	resize(c, m->wx, m->wy, m->ww - 2 * c->bw, (n == 1 ? m->wh : mh) - 2 * c->bw, False);
 	if(--n == 0)
@@ -21,6 +22,7 @@
 	if(w < bh)
 		w = m->ww;
 	for(i = 0, c = nexttiled(c->next); c; c = nexttiled(c->next), i++) {
+		c->bw = borderpx;
 		resize(c, x, y, /* remainder */ ((i + 1 == n)
 		       ? m->wx + m->ww - x - 2 * c->bw : w - 2 * c->bw), h - 2 * c->bw, False);
 		if(w != m->ww)
diff -aur a/bstackhoriz.c b/bstackhoriz.c
--- a/bstackhoriz.c	2010-08-18 01:17:14.608668919 -0400
+++ b/bstackhoriz.c	2010-08-18 01:22:01.825314185 -0400
@@ -9,6 +9,7 @@
 		return;
 	/* master */
 	c = nexttiled(m->clients);
+	c->bw = (n > 1 ? borderpx : 0);
 	mh = m->mfact * m->wh;
 	resize(c, m->wx, m->wy, m->ww - 2 * c->bw, (n == 1 ? m->wh : mh) - 2 * c->bw, False);
 	if(--n == 0)
@@ -22,6 +23,7 @@
 	if(h < bh)
 		h = m->wh;
 	for(i = 0, c = nexttiled(c->next); c; c = nexttiled(c->next), i++) {
+		c->bw = borderpx;
 		resize(c, x, y, w - 2 * c->bw, /* remainder */ ((i + 1 == n)
 		       ? m->wy + m->wh - y - 2 * c->bw : h - 2 * c->bw), False);
 		if(h != m->wh)
diff -aur a/dwm.c b/dwm.c
--- a/dwm.c	2011-12-20 10:44:53.639814729 -0500
+++ b/dwm.c	2011-12-20 11:04:43.703184812 -0500
@@ -1140,7 +1140,7 @@
 	/* only fix client y-offset, if the client center might cover the bar */
 	c->y = MAX(c->y, ((c->mon->by == c->mon->my) && (c->x + (c->w / 2) >= c->mon->wx)
 	           && (c->x + (c->w / 2) < c->mon->wx + c->mon->ww)) ? bh : c->mon->my);
-	c->bw = borderpx;
+	c->bw = 0;
 
 	wc.border_width = c->bw;
 	XConfigureWindow(dpy, w, CWBorderWidth, &wc);
@@ -1153,8 +1153,10 @@
 	grabbuttons(c, False);
 	if(!c->isfloating)
 		c->isfloating = c->oldstate = trans != None || c->isfixed;
-	if(c->isfloating)
+	if(c->isfloating) {
+		c->bw = borderpx;
 		XRaiseWindow(dpy, c->win);
+	}
 	attach(c);
 	attachstack(c);
 	XMoveResizeWindow(dpy, c->win, c->x + 2 * sw, c->y, c->w, c->h); /* some windows require this */
@@ -1199,8 +1201,10 @@
 			n++;
 	if(n > 0) /* override layout symbol */
 		snprintf(m->ltsymbol, sizeof m->ltsymbol, "[%d]", n);
-	for(c = nexttiled(m->clients); c; c = nexttiled(c->next))
+	for(c = nexttiled(m->clients); c; c = nexttiled(c->next)) {
+		c->bw = 0;
 		resize(c, m->wx, m->wy, m->ww - 2 * c->bw, m->wh - 2 * c->bw, False);
+	}
 }
 
 void
@@ -1716,11 +1720,13 @@
 		mw = m->ww;
 	for(i = my = ty = 0, c = nexttiled(m->clients); c; c = nexttiled(c->next), i++)
 		if(i < m->nmaster) {
+			c->bw = (n > 1 ? borderpx : 0);
 			h = (m->wh - my) / (MIN(n, m->nmaster) - i);
 			resize(c, m->wx, m->wy + my, mw - (2*c->bw), h - (2*c->bw), False);
 			my += HEIGHT(c);
 		}
 		else {
+			c->bw = borderpx;
 			h = (m->wh - ty) / (n - i);
 			resize(c, m->wx + mw, m->wy + ty, m->ww - mw - (2*c->bw), h - (2*c->bw), False);
 			ty += HEIGHT(c);
