--- PKGBUILD.orig	2011-07-26 21:20:30.240855937 -0400
+++ PKGBUILD	2011-07-26 21:37:26.345351226 -0400
@@ -12,6 +12,7 @@ arch=('i686' 'x86_64')
 license=('MIT')
 options=(zipman)
 depends=('libx11' 'libxinerama')
+patchdir="$HOME/src/community/dwm"
 install=dwm.install
 source=(http://dl.suckless.org/dwm/dwm-$pkgver.tar.gz
 	config.h
@@ -22,12 +23,19 @@ md5sums=('2799f885c05817ca112d521bb247f7
 
 build() {
   cd $srcdir/$pkgname-$pkgver
-  cp $srcdir/config.h config.h
+  cp $patchdir/config.h config.h
+  cp $patchdir/config.mk config.mk
+
   sed -i 's/CPPFLAGS =/CPPFLAGS +=/g' config.mk
   sed -i 's/^CFLAGS = -g/#CFLAGS += -g/g' config.mk
   sed -i 's/^#CFLAGS = -std/CFLAGS += -std/g' config.mk
   sed -i 's/^LDFLAGS = -g/#LDFLAGS += -g/g' config.mk
   sed -i 's/^#LDFLAGS = -s/LDFLAGS += -s/g' config.mk
+
+  patch -p1 < $patchdir/bstack.diff || return 1
+  patch -p1 < $patchdir/one_visible_noborder.diff || return 1
+  patch -p1 < $patchdir/title_normal.diff || return 1
+
   make X11INC=/usr/include/X11 X11LIB=/usr/lib/X11
 }
 
